udata/core/reuse/models.py

```diff
@@ -163,6 +169,49 @@ def json_ld(self):
             result['author'] = author
 
         return result
+    
+    @property
+    def views_count(self):
+        return self.metrics['views']
+    
+    @property
+    def datasets_count(self):
+        return len(self.datasets)
+
+    def count_discussions(self):
+        from udata.models import Discussion
+        self.metrics["discussions"] = Discussion.objects(subject=self, closed=None).count()
+        self.save()
+
+    def count_issues(self):
+        from udata.models import Issue
+        self.metrics["issues"] = Issue.objects(subject=self, closed=None).count()
+        self.save()
+
+    def count_followers(self):
+        from udata.models import Follow
+        self.metrics["followers"] = Follow.objects.followers(self).count()
+        self.save()
+    
+    @classmethod
+    def get_metrics_keys(cls):
+        return [
+            "discussions",
+            "issues",
+            "datasets",
+            "followers",
+            "views",
+        ]
+
+    @property
+    def get_metrics(self):
```

Comment:
use `get_metrics_keys()` here

Reply:
well, not really possible here 🤔 this pattern (keys duplication) is broken. we should be able to find a better way 🌈